server/app/controllers/geo/AddressSuggestionJsonSerializer.java

```diff
@@ -89,7 +89,8 @@ private <T> T deserializeJson(String json, Class<T> clazz) {
     try {
       return mapper.readValue(json, clazz);
     } catch (JsonProcessingException e) {
-      throw new BadRequestException("Error deserializing " + clazz.toString());
+      throw new BadRequestException(
```

Comment:
`BadRequestException` doesn't take an exception so just appending the message to the string as it was getting lost making for difficult debugging. I'd change the exception but that will trickle out more than I want this PR to deal with.